generate_stats.py

```diff
@@ -1,44 +1,81 @@
-# This file is the script that users will actually run to generate the full set of statistics for a file
+#This file is the script that users will actually run to generate the full set of statistics for a file
 
 # set the path to find the current installation of pyMOAB
 import sys
 sys.path.append('/opt/tljh/user/lib/moab/lib/python3.6/site-packages/pymoab-5.1.0-py3.6-linux-x86_64.egg')
 from pymoab import core, types
 from pymoab.rng import Range
-
 # import the new module that defines each of the functions
 import dagmc_stats
+import argparse
+
+
+
+
 
-def report_stats(entity_ranges):
+def report_stats(stats, verbose):
     """
     Method to print a table of statistics.
     
     inputs
     ------
-    entity_ranges : a dictionary with one entry for each entity type that is a Range of handles to that type
+    stats : a dictionary with information about certain statistics for a model
     """
+    
+    if verbose: #if the user wants verbosity, print with more words
+        for native_type, native_range in stats['Native Ranges'].items():
+            print("There are {} entities of type {}.".format(native_range.size(), native_type))
+        for entityset_type, entityset_range in stats['EntitySet Ranges'].items():
+            print("There are {} entities of type {}.".format(entityset_range.size(), entityset_type))
+    else: #or, print with minimal words
+        for native_type, native_range in stats['Native Ranges'].items():
+            print("Type {}: {}".format(native_type, native_range.size()))
+        for entityset_type, entityset_range in stats['EntitySet Ranges'].items():
+            print("Type {}: {}".format(entityset_type, entityset_range.size()))
+        
+            
 
-    for entity_type, eh_range in entity_ranges.items():
-        print("There are {} entities of type {}.".format(eh_range.size(),entity_type))
-
+def collect_statistics(my_core, root_set):
+    """
+    Collects statistics for a range of different areas
+    
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    root_set : the root set for a file
+    
+    outputs
+    -------
+    stats : a dictionary containing statistics for a variety of different areas
+ 
+    """
+    stats = {}
+    dagmc_tags = dagmc_stats.get_dagmc_tags(my_core)
+    entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
+    native_ranges = dagmc_stats.get_native_ranges(my_core, root_set, entity_types)     # get Ranges of various entities
```

Comment:
Can assign directly to the dictionary:

```suggestion
    stats['Native Ranges'] = dagmc_stats.get_native_ranges(my_core, root_set, entity_types)     # get Ranges of various entities
```

Reply:
similarly for other variables